source/beamline/IDEAS/IDEASMonochromatorControl.cpp

```diff
@@ -27,7 +27,7 @@ along with Acquaman.  If not, see <http://www.gnu.org/licenses/>.
 
 
 IDEASMonochromatorControl::IDEASMonochromatorControl(QObject *parent)
-//: AMPVwStatusControl("Energy", "BL08B2-1:Energy:EV:fbk", "BL08B2-1:Energy:EV", "SMTR1608-9-B20-05:brag.DMOV", "BL08B2-1:Energy:stop", parent, AMCONTROL_TOLERANCE_DONT_CARE, 2.0, new AMControlStatusCheckerDefault(0), 1, "Mono Energy")
```

Comment:
I'm not sure why this comment is still here, so I updated the PV name reference in it and left it until I have time to assess it.